raiden/api/rest.py

```diff
@@ -646,7 +647,7 @@ def open(
                 return api_error(errors=str(e), status_code=HTTPStatus.PAYMENT_REQUIRED)
             except (NonexistingChannel, UnknownTokenAddress) as e:
                 return api_error(errors=str(e), status_code=HTTPStatus.BAD_REQUEST)
-            except (DepositOverLimit, DepositMismatch) as e:
+            except (DepositOverLimit, DepositMismatch, ChannelNotOpenError) as e:
```

Comment:
I am not sure about handling it here. Since this is right after opening in the same function I don't see how a race with the channel being closed can happen here.

But I opted to handle it here too just to be safe.